test-cases/javascript/state-changing-requests/js-delete.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CrawlMaze - Testbed for Web Crawlers - form tag</title>
+<h1> DELETE method from JavaScript </h1>
+<form id="form">
+  <input type="text" name="test1" value="test">
+  <input type="text" name="test2" value="test">
```

Comment:
We want to have as minimal test cases as possible so I think you can remove the input fields here and in other cases.

Reply:
Thanks, done :)